docs/concepts/why_splinter.md

```diff
@@ -0,0 +1,285 @@
+  layers by implementing a few simple Rust traits.
+
+## <a name="bookmark-contracts-share-state"></a> Smart Contracts That Can Share State
+
+[Sawtooth Sabre](https://github.com/hyperledger/sawtooth-sabre), which is
+included in the Scabbard service, implements **in-ledger smart contracts** that
+are executed in a WebAssembly (WASM) virtual machine. Smart contracts can be
+written in any language that can be compiled to WASM.
+
+Scabbard's smart contracts are stored in state and executed by each participant
+node in the circuit, rather than on a separate node or virtual machine. This
+state is shared with all participants on the circuit, but the services for that
+circuit ensure that the state is invisible to other Splinter nodes (those not
+participating in the circuit).
+
+Most importantly, a smart contract can **share state with other smart
+contracts**. Each smart contract manages its own area of state (its own
+_namespace_), but is not restricted to that state. A smart contract can access
+another smart contract's state, as controlled by Sabre permissions. Splinter's
+design for sharing state simplifies the complex approach of other blockchain
+platforms, where sharing state means embedding one smart contract into another.
+
+Sabre's smart contracts are compatible with [Hyperledger
+Sawtooth](https://github.com/hyperledger/sawtooth-core). You can write a
+Sawtooth transaction processor (Sawtooth's version of a smart contract) in Rust,
+then convert it to a Sabre smart contract that can be used for a Splinter
+circuit.
+
+## <a name="bookmark-quick-dev-startup"></a> Quick Startup for Application Development
+
+Splinter includes several APIs for application development:
+
+* **Service API** for building custom services to send and manage opaque
+  messages between nodes on a circuit. The Scabbard service handles consensus,
+  smart contracts, and transaction processing (with the two-phase commit
+  agreement protocol, Sawtooth Sabre, and Hyperledger Transact, respectively).
+  However, Splinter is designed to support other smart contract engines and
+  transaction processing software.
+
+* **Consensus API** for building custom consensus protocols. The Scabbard
+  service currently includes the multi-party [two-phase
+  commit](https://en.wikipedia.org/wiki/Two-phase_commit_protocol) agreement
+  protocol instead of a consensus protocol. The consensus API makes it easy to
+  add other types of consensus, such as Byzantine Fault Tolerant (BFT) or Proof
+  of Work (PoW).
+
+* **Custom REST APIs** for extending services to external tools. For example,
+  the Scabbard service uses this API to expose endpoints for smart contract
+  submission.
+
+Splinter provides a growing list of optional features that are needed by many
+applications, so that application developers don't have to reimplement this
+common functionality. For example, the application authorization handler manages
```

Comment:
Might want to put "application authorization handler" in bold to visually indicate that it's specific thing in splinter. Same for Biome below.

Reply:
Good idea. Done.